### DISCUSSION TITLE
how to check if the traveler is within the list of travelers?

### DISCUSSION THREAD

**Discussion by valiantlynx** [pocketbase/pocketbase#3720] [Q&A] [answered]
i'm making a trips management system. and there can be multiple travelers on the same trip.

to create a trip there hase to be a traveller id and it has to be the same as the one being created.

ive tried `@request.auth.id != ""  &&  @request.data.travelers ?= @request.auth.id`

but i get error. 
i thought the ?= will check if the id @request.auth.id  is one of the arrayof ids in @request.data.travelers

how do i check if the id if one of the ids in the array in pocketbase api rules?

thanks in advance

**Comment by ganigeorgiev** [comment, marked as answer]
`@request.data.travelers` will correspond to the submitted data value (in your case probably an array).
If `travelers` is a multiple relation field and you want to check that at least one of its submitted values is equal to the current authenticated user, you can use dot-notation to apply a condition on each traveler's field value, aka:

```js
@request.auth.id != ""  &&  @request.data.travelers.id ?= @request.auth.id
```

**Comment by valiantlynx** [comment]
nice i would not have thought of that thanks. that worked fine
